### ISSUE TITLE
problem - nie możesz korzystać z tej metody Publicznego API.

### ISSUE THREAD

**Issue by rradek251** [allegro/allegro-api#7033]
Występuje problem: 
 - [ ] z zasobem REST API / with a REST API resource
 - [ ] ze środowiskiem produkcyjnym / with the production environment
 - [ ] ze środowiskiem testowym / with the test environment
 - [ ] z logowaniem lub autoryzacją / with login or authorization
 - [ ] na portalu https://developer.allegro.pl/ / at https://developer.allegro.pl/en/
 - [ ] inny / other

:beetle: **Opis / Description**

**Comment by MaciejFrackowiak** [comment]
Proszę podaj więcej szczegółów, abyśmy mogli wyjaśnić wątpliwości.

**Comment by rradek251** [comment]
Witam wystawiłem produkt na Allegro poprzez platformę do dropshippingi jako
osoba prywatne, działalność nie zarejestrowana bez firmy i wyskakuje mi tam
komunikat jak w tytule ze nie można korzystać z tej metody publicznego API

W dniu czw., 4.05.2023 o 06:52 MaciejFrackowiak ***@***.***>
napisał(a):

**Comment by MaciejFrackowiak** [comment]
W tym wypadku musisz się skontaktować z dostawcą wykorzystywanego oprogramowania, ponieważ tylko tak uzyskasz informację do jakiego zasobu nie posiadasz dostępu.

**Comment by rradek251** [comment]
Właśnie kazali kontaktować się z Allegro, a może Pan wie jakie platformy
dropshoppingowe współpracują z Allegro?

W dniu czw., 4.05.2023 o 10:16 MaciejFrackowiak ***@***.***>
napisał(a):

**Comment by MaciejFrackowiak** [comment]
Nie, nie prowadzimy takiego rejestru. Aby korzystać z API i wystawiać przez nie oferty musisz posiadać konto firmowe. Korzystając z konta zwykłego takiej oferty nie wystawisz.

**Comment by rradek251** [comment]
No to trochę dziwne zasady ma Allegro bo według Polskiego prawa nie
rejestrując się jako firma mogę dokonać sprzedaży na kwote 50% najniższej
krajowej a Allegro to uniemożliwia …

W dniu czw., 4.05.2023 o 10:36 MaciejFrackowiak ***@***.***>
napisał(a):

**Comment by MaciejFrackowiak** [comment]
Ależ skąd - nadal możesz wystawiać oferty przez formularz wystawiania dostępny na stronie Allegro.
